src/components/projects/projects-table/projects-table-filter/projects-table-filter.module.css

```diff
@@ -32,6 +32,16 @@
   border-color: #ef4345 !important;
 }
 
+.clearButton {
+  float: left;
+  background-color: #ef4345 !important;
+  border-color: #ef4345 !important;
+}
+
 .applyButton:focus {
-  box-shadow: 0 0 0 .2rem rgba(239, 67, 69, 0.5) !important;
+  box-shadow: 0 0 0 0.2rem rgba(239, 67, 69, 0.5) !important;
+}
+
+.clearButton:focus {
+  box-shadow: 0 0 0 0.2rem rgba(239, 67, 69, 0.5) !important;
```

Comment:
We should probably consolidate all of our theme styling and colors so that they are easier to refer to when we are styling components. @jamescd18 this could be a new epic for refactoring all our styling to have a theme provider or something like that as our single source of styling and then implementing that into all our components so that if we do decide to change our theme styles, it's much easier to change in one place than in every single component.